tool/colors/flutter/colors_material.dart

```diff
@@ -1,3 +1,5 @@
+/// This file was downloaded by update_colors.dart.
```

Comment:
should probably use a double slash comment

Reply:
Done!